### DISCUSSION TITLE
Modern Farm wireframe flow chart

### DISCUSSION THREAD

**Discussion by DavidEdwardDarden** [nss-day-cohort-47/C47-Discussions#44] [Discuss a topic]
My name is David Darden.

Have you searched the other help tickets? Yes

Describe your issue: Does anyone have a flow chart or wireframe of the Modern Farm exercise. I am having trouble understanding broadly what the program is supposed to do.

What have you googled? Yes, but nothing helpful came up.

What have you tried? Tell us everything you have tried. I tried drawing my own flow chart/wireframe.

**Comment by EthanMathis** [comment]
I didn't actually wireframe this project out because most of it is given to you directly. So, realistically, you're just modifying an already existing (relatively complex) application. This whole project is kind of weird though. It runs in the terminal and was rather unintuitive, for me at least. If you're trying to figure out how to get it all set up and running feel free to holler at me on slack and I'll help you however I can :)

**Reply by Chadwell-Clark** [reply]
I will be glad to help also David. It reinforces the concepts we used in Chapter one, specifically Martin's Aquarium. It is mostly about using modules to export and import functions that do one task, manipulating objects to extract their data and then doing something with the data.
The instructions are pretty much a flow chart but you could draw one up if you wanted. 
Hit me up if I can help.
